### ISSUE TITLE
Default formatting is not the one showing on pypi

### ISSUE THREAD

**Issue by LarsStavholm** [vaab/gitchangelog#25]
I've used the default `gitchangelog.rc.reference` to generate a change log for my project. However, I didn't get the (nicer) formatting expected as seen on the pypi page. Although the pypi page says: And the rendered full result is directly used to generate the HTML webpage of the changelog of the PyPI page.", I don't get the same result. In my output there's "Change-Id.." and extra newlines, like:

```
Changelog
=========

0.3.0 (2015-03-08)
------------------

- Doco update. [Lars Stavholm]

  Change-Id: Ib81c683769d31a1b4923df0f2ec937b2ad22f04a


- Rearranged rack files. [Lars Stavholm]

  Change-Id: I2a31001299c283d23b69db1622aec633e891abaa


- Updated. [Lars Stavholm]

  Change-Id: I6aa8fd0c479617257f6325b1ca16366ec582bbe3

```

**Comment by vaab** [comment]
Ok, for some reason you have extra tags in your commits. Supporting this case is something I'm ready to implement in ``gitchangelog``. Can you provide me with a sample ``git log`` of 2 or 3 representative commit ?

If you are not a git guru::

    git log -n 3

... might just give the output I need.

This seems to be related to using ``git-review``, probably because you are using ``Gerrit``.

As a possible a quick work-around, you might want to try to modify your ``.gitchangelog.rc`` add use this filter::

    body_process = ReSub(r'(\n[A-Z]\w+(-\w+)*: .*)+$', r'') | strip

If this is okay with you I'll probably set it as the default in the reference file.

**Comment by LarsStavholm** [comment]
First of all: brilliant tool!

No, I'm not a git guru of any kind:

```
% git log -n 3
commit e1b66190cf9fc222761981c4fe8e844157b78cb2
Author: Lars Stavholm <lars.stavholm@ericsson.com>
Date:   Sun Mar 8 12:59:53 2015 +0100

    Doco update.
    
    Change-Id: Ib81c683769d31a1b4923df0f2ec937b2ad22f04a

commit 4549a0086a8eeaff6fdfe1160e026dc414a11785
Author: Lars Stavholm <lars.stavholm@ericsson.com>
Date:   Sun Mar 8 07:08:01 2015 +0100

    Rearranged rack files.
    
    Change-Id: I2a31001299c283d23b69db1622aec633e891abaa

commit f0bdeb4883bc7a2a44b37f9503bb1f1c85d489d7
Author: Lars Stavholm <lars.stavholm@ericsson.com>
Date:   Sun Mar 8 07:07:31 2015 +0100

    Updated.
    
    Change-Id: I6aa8fd0c479617257f6325b1ca16366ec582bbe3
```
Yes, we're using gerrit.

The new `body_process` definition didn't help a whole lot:
```
% gitchangelog
Changelog
=========

0.3.0 (2015-03-08)
------------------

- Doco update. [Lars Stavholm]

  Change-Id: Ib81c683769d31a1b4923df0f2ec937b2ad22f04a

- Rearranged rack files. [Lars Stavholm]

  Change-Id: I2a31001299c283d23b69db1622aec633e891abaa

- Updated. [Lars Stavholm]

  Change-Id: I6aa8fd0c479617257f6325b1ca16366ec582bbe3

[snip]
```

**Comment by vaab** [comment]
Thanks, this should be it:

    body_process = ReSub(r'((^|\n)[A-Z]\w+(-\w+)*: .*)+$', r'') | strip

**Comment by LarsStavholm** [comment]
And it is!

We haven't started using the keywords `chg`, `new`, etc, but it looks really good already, thank you
```
Changelog
=========

0.3.0 (2015-03-08)
------------------

- Doco update. [Lars Stavholm]

- Rearranged rack files. [Lars Stavholm]

- Updated. [Lars Stavholm]

- Cosmetic. [Lars Stavholm]

- Added spare parts from 81-Z5. [Lars Stavholm]

- Cosmetic. [Lars Stavholm]

- Re-added. [Lars Stavholm]

- Rubbish. [Lars Stavholm]

- Closed #162. [Lars Stavholm]

- Bug fix: added missing weballoc templates dir. [Lars Stavholm]

- Bug fix: added missing rack files in packaging. [Lars Stavholm]

0.2.0 (2015-03-03)
------------------

- Doco update. [Lars Stavholm]

- Spare parts list. [Elias Johnstone]

- Improved scanning and loading of database. Tweaked first page. [Lars
  Stavholm]
[snip]
```

How can I remove the extra newlines? I'd like something like:
```
Changelog
=========
0.3.0 (2015-03-08)
------------------
- Doco update. [Lars Stavholm]
- Rearranged rack files. [Lars Stavholm]
- Updated. [Lars Stavholm]
- Cosmetic. [Lars Stavholm]
- Added spare parts from 81-Z5. [Lars Stavholm]
- Cosmetic. [Lars Stavholm]
- Re-added. [Lars Stavholm]
- Rubbish. [Lars Stavholm]
- Closed #162. [Lars Stavholm]
- Bug fix: added missing weballoc templates dir. [Lars Stavholm]
- Bug fix: added missing rack files in packaging. [Lars Stavholm]

0.2.0 (2015-03-03)
------------------
- Doco update. [Lars Stavholm]
- Spare parts list. [Elias Johnstone]
- Improved scanning and loading of database. Tweaked first page. [Lars Stavholm]
[snip]
```
